scapy/utils6.py

```diff
@@ -783,12 +779,14 @@ def in6_getscope(addr):
     """
     Returns the scope of the address.
     """
-    if in6_isgladdr(addr) or in6_isuladdr(addr):
+    if in6_isgladdr(addr):
```

Comment:
According to https://tools.ietf.org/html/rfc4193#section-3.3 the scope is global. This function should not be modified.